src/pages/jquery/jquery-effects-show-method/index.md

```diff
@@ -28,6 +28,14 @@ $("#title").show( "slow", function() {
   ```
 * More options exist, please refer to the official website for further details. 
 
+### .toggle() method
```

Comment:
I think this should be a separate article. If there is one dedicated to "show" and "hide", I would add this as a separate article.